app/public/js/movie.js

```diff
@@ -1,4 +1,4 @@
-var decision_time = 15;
+var decision_time = 12;
```

Comment:
Or if you want to disable the timer entirely because you want to close #37 then do so.